shared/locales/media/media.en-US.properties

```diff
@@ -0,0 +1,13 @@
+# Frame descriptions
+imageDescription = {{orientation}} image, %I:%M %p, %B %e, %Y
+imageDescriptionNoTimestamp = {{orientation}} image
+videoDescription = {{orientation}} video, %I:%M %p, %B %e, %Y
+videoDescriptionNoTimestamp = {{orientation}} video
+
+orientationPortrait = portrait
+orientationLandscape = landscape
+
+# Video controls
+playbackPlay.ariaLabel=Play
+playbackPause.ariaLabel=Pause
+playbackFullscreen=Fullscreen
```

Comment:
Yes, oops. Looks like this patch got squashed with a different one. I'll rebase and upload a cleaner one.